salt/states/chocolatey.py

```diff
@@ -86,7 +86,7 @@ def installed(name, version=None, source=None, force=False, pre_versions=False,
            'comment': ''}
 
     # Determine if the package is installed
-    if name not in __salt__['cmd.run']('choco list --local-only'):
+    if name not in __salt__['cmd.run']('choco list --local-only --limit-output'):
```

Comment:
Apparently fixing the merge conflict hid the comments, I considered opening a new issue about this version checking but wasn't sure if that made sense. @Ch3LL what do you think?